content/concepts/products.md

```diff
@@ -141,7 +142,8 @@ In this formula:
  - `LOCALE_CODE` is the code of a locale when the attribute is localizable, should be equal to `null` otherwise. [Check some examples here.](#the-locale-and-scope-format)
  - `CHANNEL_CODE` is the code of a channel when the attribute is scopable, should be equal to `null` otherwise. [Check some examples here.](#the-locale-and-scope-format)
  - `DATA_INFORMATION` is the value stored for this attribute for this locale (if attribute is localizable) and this channel (if the attribute is scopable). Its type and format depend on the attribute type. [Check some examples here.](#the-data-format)
-
+ - `LINKED_DATA` containing labels if the attribute is an attribute option (only available in Serenity). [Check some examples here.](#the-linked_data-format)
```

Comment:
Attribute can't "be" an attribution option. It can "have". But I would rephrase this way:
```suggestion
 - `LINKED_DATA` containing labels if the attribute is a simple or multi select. [Check some examples here.](#the-linked_data-format) This property if for now, only available in Serenity.
```